test/contracts-network/colony-expenditure.js

```diff
@@ -304,16 +370,6 @@ contract("Colony Expenditure", (accounts) => {
 
       await colony.finalizeExpenditure(expenditureId, { from: ADMIN });
     });
-
-    it("should not allow admins to update payouts", async () => {
-      await colony.finalizeExpenditure(expenditureId, { from: ADMIN });
-      await checkErrorRevert(colony.setExpenditurePayout(expenditureId, SLOT0, token.address, WAD, { from: ADMIN }), "colony-expenditure-not-active");
-    });
-
-    it("should not allow admins to update skills", async () => {
-      await colony.finalizeExpenditure(expenditureId, { from: ADMIN });
-      await checkErrorRevert(colony.setExpenditureSkill(expenditureId, SLOT0, 1, { from: ADMIN }), "colony-expenditure-not-active");
-    });
```

Comment:
I feel like tests involving deprecated functions should only be removed when the function is removed.

Reply:
Fair, but I'll restore these using the new array-based version.